nominees/data-observation-toolkit.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Data Observation Toolkit",
+  "aliases": [
+    "DOT"
+  ],
+  "description": "DOT is a toolkit that can be used to monitor data as it's collected and enters a database to flag problems with data integrity, data quality, protocols, outliers and other scenarios that might need attention",
+  "website": "https://github.com/datakind/Data-Observation-Toolkit",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/datakind/Data-Observation-Toolkit/blob/main/LICENSE.md"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "In 2019, the United Nations Statistical Commission stated: “Every misclassified or unrecorded death is a lost opportunity to ensure other mothers and babies do not die in the same way. When it comes to health, better data can be a matter of life and death.”  DataKind developed the Data Observation Toolkit (DOT) in response to a gap in digital health solutions space to monitor health data generated by other digital tools for problems with data integrity, quality and other scenarios that might need attention. The toolkit aims to provide easier access to data monitoring without the need for deep technical skills. ",
+      "evidenceURL": "https://www.datakind.org/blog/empowering-health-workers-and-community-health-systems"
+    }
+  ],
+  "sectors": [
+    "Health"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositories": [
+    {
+      "name": "datakind / Data-Observation-Toolkit",
```

Comment:
Could we update that this to just have the repository name: Data-Observation-Toolkit

Reply:
Fixed in commit 8a6ecec